### ISSUE TITLE
Link text is not proper for one of the skip links on all English pages

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#113]
URL: https://hcdev.openplus.ca/en
User Agent: Firefox (v92)
Language: English

Issue Description: 
The skip link that allows users to jump to the "About government" section doesn't have a proper link text. Currently, it is labelled as Skip to “About this site”.  

![image](https://user-images.githubusercontent.com/12256067/137023741-4d462f40-4782-45e5-9577-969dd992d0d0.png)

Steps to reproduce:

1. Open the URL provided above
2. Navigate the page using the "Tab" key

Expected results: To have the proper link text and to be consistent with Canada.ca template, it should be labelled as Skip to “About government”.

**Comment by ndolter** [comment]
The skip links are no longer getting displayed when tabbing.

**Comment by echouin** [comment]
@sarasuri OP confirms fixed.

**Comment by ndolter** [comment]
Validate that the 'Skip to "About government" link has now been corrected in both languages, however, noted that the French page 'Aller au contenu principal' skip link does not match to that of the Canada.ca template which is 'Passer au contenu principal'.
![image](https://user-images.githubusercontent.com/68117281/137513711-0c48d6a9-9e3f-4d70-b450-8df14a2938bb.png)

**Comment by echouin** [comment]
@ndolter From OP: 
I'm not seeing this

﻿on ﻿https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html﻿ I see
﻿﻿<a class="wb-sl" href="#wb-info">Passer à « Au sujet du gouvernement »</a>

﻿and on ﻿https://hcdev.openplus.ca/fr﻿ I see
﻿﻿<a class="wb-sl" href="#wb-info">Passer à « Au sujet du gouvernement »</a>

**Comment by mmiron-HC** [comment]
@echouin this is still an issue. the french has the word Aller instead of passer
![image](https://user-images.githubusercontent.com/65728700/138887247-ed30d93d-6b1f-4c71-9224-44f2e8ed658f.png)

**Comment by echouin** [comment]
@mmiron-HC requested OP fix.

**Comment by echouin** [comment]
This is now resolved:
![image](https://user-images.githubusercontent.com/88099454/148393073-c4fb2a2c-759e-4d13-b7f9-ada2662974e0.png)

**Comment by mmiron-HC** [comment]
Validated that both the French and English pages have the correct text.

**Comment by ndolter** [comment]
Validated that both the French and English pages have the correct text.
